docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,254 @@ grand_parent: "General"
+
+```html
+<!-- BPMN diagram container -->
+<div id="canvas"></div>
+```
+
+Here, we've given an ID to the root DOM element since we'll be using this ID to select this component and pass to bpmn-js later on.
+
+## Including the Stylesheet
+
+We should include [diagram-js](/general/getting-started/diagram-js) CSS and [BPMN Icon Font](https://github.com/bpmn-io/bpmn-font) in order to ensure that everything looks good. There're several ways we can achieve that:
+
+We can include the CSS via `link` tag in our HTML file. Note that these CSS files are shipped with with the bpmn-js distribution under [dist/assets folder](https://unpkg.com/browse/bpmn-js/dist/assets/).
+
+```html
+<link rel="stylesheet" href="bpmn-js/dist/assets/diagram-js.css" />
+<link rel="stylesheet" href="bpmn-js/dist/assets/bpmn-font/css/bpmn.css" />
+```
+
+If we've install the bpmn-js via [NPM](#including-via-npm), we can additionally include these CSS files using `import` statement in our Javascript file:
+
+```javascript
+import 'bpmn-js/dist/assets/diagram-js.css';
+import 'bpmn-js/dist/assets/bpmn-font/css/bpmn-embedded.css';
+```
+
+## Alternatives to Use bpmn-js
+
+We can use bpmn-js either as:
+
+* [a modeler](#using-bpmn-js-as-modeler) to create BPMN 2.0 diagrams
+* [a viewer](#using-bpmn-js-as-viewer) to view BPMN 2.0 diagrams
+* [a vavigated viewer](#using-bpmn-js-as-viewer) to view BPMN 2.0 diagrams and navigate through them
+
+In this section we'll explain the use cases of those and how to get them running in any web application.
+
+### Using bpmn-js as Viewer
+
+bpmn-js may be used as a viewer. In that case the user would not be able to interact with the diagram to modify the existing BPMN 2.0 XML, but would only be able to view it. In addition to viewer, bpmn-js offers a navigated viewer which enables [move-keyboard](/modules/common-modules/keyboard-move), [move-canvas](/modules/common-modules/move-canvas) and [zoom-scroll](/modules/common-modules/zoom-scroll) [modules](/modules/adding-customizing-removing) by default.
+
+#### Including the Pre-packaged Version
+
+For the pre-packaged version, we should include the bpmn-js viewer via HTML script tag:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-viewer.production.min.js"></script>
+```
+
+or for the navigated viewer:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-navigated-viewer.production.min.js"></script>
+```
+
+#### Including via NPM
+
+First, we should install bpmn-js via NPM:
+```
+npm install bpmn-js
+```
+
+We should then import the BPMN viewer as an ES module:
+
+```javascript
+import Viewer from 'bpmn-js/lib/Viewer';
+```
+
+or for the navigated viewer:
+
+```javascript
+import NavigatedViewer from 'bpmn-js/lib/NavigatedViewer';
+```
+
+#### Viewing the BPMN 2.0 XML
```

Comment:
Sounds like this is only about the viewer. I'd change the title to something like _Import a BPMN 2.0 Diagram_.

Reply:
Done 👍